sig-architecture/api-review-process.md

```diff
@@ -30,39 +32,37 @@ Ideally, those whose API review priority is shifted in a release-impacting way w
 
 # What APIs need to be reviewed?
 
-* What are the kind of reviews?
-
-    * **mandatory**: The changer is expected to submit the changes to API review, and the change is blocked on the API reviewer approval 
-
-    * **voluntary**: Changer can request a review, or a 3rd party can nominate the API for review.  The review comments are recommendations.
+## Mandatory
+The changer is expected to submit the changes to API review, and the change is blocked on the API reviewer approval.  As a general rule of thumb, anything that is considered to be part of "core Kubernetes" requires a mandatory API review.  The list below outlines what are considered Kubernetes core APIs:
 
-* Which projects are in scope?
+* All API implementations (including alpha versions) that are part of core Kubernetes must be reviewed, including CRDs, so user experience across the Kubernetes ecosystem is consistent.
 
-    * github.com/kubernetes/kubernetes, and any kubernetes-* org projects it depends on are required to be reviewed.   (mandatory)
+* github.com/kubernetes/kubernetes, and any kubernetes-* org projects it depends on are required to be reviewed.
 
-    * Any kubernetes-style API that uses a *.k8s.io or *.kubernetes.io name, e.g. "storage.k8s.io" (mandatory)
+* Kubernetes-style API that uses a *.k8s.io or *.kubernetes.io name, e.g. "storage.k8s.io".
 
-    * "Critical" or other “highly-integrated” APIs, such as our extension points in the node, apiserver, and controllers as defined by SIG Architecture [TBD].  (mandatory)
+* "Critical" or other “highly-integrated” APIs, such as our extension points in the node, apiserver, and controllers.  This includes CSI, CNI, CRI, and CPI.
 
-    * any other github projects in the kubernetes-* orgs (informational, for now - may make mandatory in future)
+## Voluntary
+Voluntary reviews apply towards non-core APIs that do not meet the [mandatory](#mandatory) requirements listed above.  Changer can request a review, or a 3rd party can nominate the API for review.  The review comments are considered recommendations.    
 
-    * Any projects that produce APIs (including CRDs) that are intended to work with kubectl and/or kube-apiserver. (informational - intent is to ensure consistent user experience across the Kubernetes ecosystem)
+* SIG sponsored CRD based APIs outside of the core that use the "*.x-k8s.io" namespace.
```

Comment:
I'm in favor of this suffix for this purpose.

Need to iron out the following before we recommend it (not sure where... maybe this doc, or maybe peer to the [repository guidelines](https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md#sig-repositories), linked from here):
* get the domain transferred to cncf/kubernetes
* indicate how sigs use it. I'd imagine sigs automatically get to use `[*.]<signame>.x-k8s.io`, and optionally can add other prefixes that make sense (e.g. sig-auth: ["authentication.x-k8s.io","authorization.x-k8s.io"], sig-cluster-lifecycle: ["cluster.x-k8s.io"], etc) to something like the [sigs.yaml](https://github.com/kubernetes/community/blob/master/sigs.yaml) file for visibility.

Reply:
(before we do that work, any concerns from @kubernetes/api-approvers on moving forward with that approach?)

Reply:
ping @jbeda re-domain.

Reply:
I'll get the domain moving now -- don't wait on me.  I'll ping Chris at CNCF.

Reply:
@jbeda please use the CNCF service desk so people other than Chris at the CNCF can pick it up and it is tracked.

Reply:
`x-` is a common pattern for extensions. I agree we should follow a well understood pattern rather than try to invent our own. So, I'm in favor.